apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs

```diff
@@ -246,6 +246,63 @@ public partial class BigtableClient
         }
     }
 
+    /// <summary>
+    /// Settings for creating <see cref="BigtableServiceApiClient"/>s.
+    /// </summary>
+    public sealed class ClientCreationSettings
```

Comment:
Let's make this nested within `BigtableClient` so we are consistent with the Pubsub clients.